extensions/amp-ad-network-adsense-impl/0.1/amp-ad-network-adsense-impl.js

```diff
@@ -244,14 +244,16 @@ export class AmpAdNetworkAdsenseImpl extends AmpA4A {
   }
 
   /** @override */
-  getAdUrl(consentState) {
+  getAdUrl(consentTuple) {
```

Comment:
getAdUrl({consentState, consentString}) will allow you to reference them directly as variables throughout without prepending with consentTuple